cypress/e2e/shared/checks.test.ts

```diff
@@ -545,4 +573,173 @@ describe('Checks', () => {
+          .invoke('text')
+          .wrap('checkIDOrig')
+        cy.getByTestID('copy-task-id')
+          .invoke('text')
+          .wrap('taskIDOrig')
+        // 2. Clone
+        cy.getByTestID('context-menu-task').click()
+        cy.getByTestID('context-clone-task').click()
+        cy.wait('@createCheck')
+        // 3. Asserts
+        cy.get<GenCheck>('@check').then(check => {
+          cy.getByTestID(`check-card ${cloneName}`).within(() => {
+            cy.getByTestID('copy-resource-id').should('not.have.text', check.id)
+            cy.getByTestID('copy-task-id').should('not.have.text', check.taskID)
+          })
+        })
+
+        cy.getByTestID('check-card--name').then(nameDivs => {
+          const names = nameDivs.toArray().map((r: any) => r.innerText)
+          expect(names.length).to.equal(2)
+          expect(names[1]).to.equal(cloneName)
+        })
+        // 3.1 Assert configure same
+        cy.contains(cloneName).click()
+        cy.getByTestID('duration-input--for').should(
+          'have.value',
+          deadmanCheck.timeSince
+        )
+        cy.getByTestID('check-levels--dropdown--button').should(
+          'contain',
+          deadmanCheck.level
+        )
+        cy.getByTestID('duration-input--stop').should(
+          'have.value',
+          deadmanCheck.staleTime
+        )
+        cy.getByTestID('status-message-textarea').should(
+          'contain',
+          deadmanCheck.statusMessageTemplate
+        )
+        cy.getByTestID('offset-options').should(
+          'have.value',
+          deadmanCheck.offset
+        )
+        cy.getByTestID('schedule-check').should(
+          'have.value',
+          deadmanCheck.every
+        )
+        // 3.2 Assert query-builder
+        cy.getByTestID('select-group--option').click()
+        cy.getByTestID(`selector-list ${bucketName}`).should(
+          'have.class',
+          'cf-list-item__active'
+        )
+        cy.getByTestID('selector-list wumpus').should(
+          'have.class',
+          'cf-list-item__active'
+        )
+        cy.getByTestID('selector-list dur').should(
+          'have.class',
+          'cf-list-item__active'
+        )
+        cy.getByTestID('selector-list bar').should('be.visible')
+        cy.getByTestID('giraffe-layer-line').should('be.visible')
+        cy.getByTestID('square-button')
+          .eq(0)
+          .click()
+        cy.getByTestID(`check-card ${cloneName}`).within(() => {
+          cy.getByTestID('context-delete-task--button').click()
+        })
+        cy.getByTestID('context-delete-task--confirm-button').click()
+        cy.getByTestID(`check-card ${cloneName}`).should('not.exist')
+        cy.getByTestID(`check-card ${deadmanCheck.name}`).should('be.visible')
+        cy.getByTestID('check-card--name').should('have.length', 1)
+      })
+    })
+
+    // TODO fill in actual test
+    it.skip('can clone a threshold check', () => {
+      // 1. create threshold over API
+    })
+  })
```

Comment:
I'd rather just remove this blank test from the PR and add it when the test is ready of that's ok

Reply:
Removed